clipper_admin/clipper_admin/docker/docker_container_manager.py

```diff
@@ -475,6 +476,19 @@ def stop_all(self, graceful=True):
             else:
                 c.kill()
 
+    # Wait for maximum 5 min.
+    @retry((docker.errors.NotFound, docker.errors.APIError, ClipperException),
+           tries=300, delay=1, backoff=1, logger=logger)
+    def _check_container_status(self, name):
+        state = self.docker_client.containers.get(name).attrs.get("State")
+        inspected = self.docker_client.api.inspect_container(name)
+        if (state is not None and state.get("Status") == "running") or \
+           (inspected is not None and inspected.get("State").get("Health").get("Status") == "healthy"):
+            return
+        else:
+            msg = "{} container is not running yet".format(name)
```

Comment:
Since we have lots of retry (max 5 minutes for each container status check), I think it is better having a little more descriptive message.

Why don't we change the msg to 
`{} container is not running yet or broken. We will try to run again. Please analyze logs if it keeps failing.`?

Reply:
Good advice. I will update the message.